### PR TITLE
Fix: Issue #1500 

### PR DESCRIPTION
**Pull request by Src0p**
Fixed the styling and rendering issue in endpoint section of the Services view.

### CODE REVIEW

File: frontend/src/components/service/Details.tsx [hunk 1]
```diff
@@ -100,8 +100,10 @@ export default function ServiceDetails() {
                       },
                       {
                         label: t('translation|Addresses'),
-                        getter: endpoint => endpoint.getAddressesText(),
-                        cellProps: { style: { width: '40%', maxWidth: '40%' } },
+                        getter: endpoint => {
+                          return endpoint.getAddressesText();
+                        },
```
Comment (joaquimrocha) [inline frontend/src/components/service/Details.tsx:105]: This change is not needed. As the code deleted does the same.

File: frontend/src/lib/k8s/endpoints.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ class Endpoints extends makeKubeObject<KubeEndpoint>('endpoint') {
   }
 
   getAddressesText() {
-    this.getAddresses().join(', ');
+    return this.getAddresses().join(', ');
```
Comment (joaquimrocha) [inline frontend/src/lib/k8s/endpoints.ts:47]: Ouch! Weird we got this one through.

**Review by joaquimrocha — CHANGES_REQUESTED** [review]
Hi @Src0p . Thanks for your PR!

I have commented on one of the changes you did.
The commit title should also be aligned to the [guidelines](https://github.com/kinvolk/contribution/blob/master/topics/git.md#authoring-prs) we use. So it should be something like:
```
frontend endpoin.ts: Actually return the addresses in getAddressesText

By mistake, a return statement was missing.
```

As part of Headlamp being now a CNCF Sandbox project, we also need every commit to be signed-off. So please add the `-s` option when committing/amending.

File: frontend/src/components/service/Details.tsx (hunk 1 above)
Comment (Src0p) [reply frontend/src/components/service/Details.tsx:105]: Thank you so much for the feedback , I am really sorry for overlooking the guidelines.

Comment (joaquimrocha) [reply frontend/src/components/service/Details.tsx:105]: No worries. We definitely encourage folks to submit PRs even if they need a bit adjustment to get them merged.
I saw you closed this PR. Of course you can reopen a new one when you have the new changes, but usually you just have to amend your commits and force push. If you need help with this, join our [Slack channel](https://kubernetes.slack.com/messages/headlamp) and I can help.

This is an important fix that I would like to get in the next version. Are you able to send us a new commit after my review?

Comment (joaquimrocha) [reply frontend/src/components/service/Details.tsx:105]: Ah, I see that there's another PR that's related to this and we should use that one. Sorry for the confusion @Src0p . I hope to see more contributions from you!

**Pull request closed without merging**